css-ui-4/Overview.bs

```diff
@@ -1740,10 +1740,9 @@ Note: The details of the aliasing mechanism is intentionally left up to the UA.
 Making ''-webkit-user-select'' a shorthand property of 'user-select'
 is known to be an effective approach,
 and new implementers should consider it.
-However UAs supporting ''-webkit-user-select'' as an alias of 'user-select' through other means exist,
-without adverse consequences to compatibility,
-so this specification allows flexibility.
-
+However, UAs supporting ''-webkit-user-select'' as an alias of 'user-select' through other means exist,
+without adverse consequences to compatibility.
+So this specification allows flexibility.
```

Comment:
this line should remain part of the previous sentence, as originally written. (the addition of a comma after 'however' is good, tho)

Reply:
Ok, added the comma back.